sky/shell/platform_view.h

```diff
@@ -26,8 +26,18 @@ class PlatformView {
     scoped_refptr<base::SingleThreadTaskRunner> ui_task_runner;
   };
 
+  struct SurfaceConfig {
+    uint8_t red_bits = 8;
+    uint8_t green_bits = 8;
+    uint8_t blue_bits = 8;
+    uint8_t alpha_bits = 8;
+    uint8_t depth_bits = 0;
+    uint8_t stencil_bits = 0;
```

Comment:
I think we do want 8 bits of stencil.  Doesn't have to be the default, but we should set it somewhere.